alf/environments/gym_wrappers_test.py

```diff
@@ -89,6 +89,8 @@ def test_framestack_all_fields(self):
     def test_framestack_partial_fields(self):
         env = self._create_env(['image', 'dict.inner_states'])
         obs = env.reset()
+        import pdb
+        pdb.set_trace()
```

Comment:
remove pdb

Reply:
done

Reply:
pdb still there?  Did you forget to push some commits to remote?